lib/mjsonwp.js

```diff
@@ -62,9 +62,13 @@ function getExpressRouter (driver) {
             jsonwpRes.status = 0; // TODO use status library
             jsonwpRes.value = driverRes;
           } catch (err) {
-            httpStatus = 500;
-            jsonwpRes.status = err.jsonwpCode;
-            jsonwpRes.value = err.message;
+            if (err instanceof errors.BadParametersError) {
+              httpStatus = 400;
```

Comment:
We are supposed to list the missing parameters in the response body in this case, no?